### PR TITLE
{tools}[foss/2023a] Ray-project v2.9.1

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/Ray-project/Ray-project-2.9.1-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'usempi': True}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('protobuf-python', '4.24.0'),
+    ('PyYAML', '6.0'),
+    ('redis-py', '5.0.1'),
+    ('hatchling', '1.18.0'),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/r/Ray-project/Ray-project-2.9.1-foss-2023a.eb:21]: please move to builddependencies

File: easybuild/easyconfigs/r/Ray-project/Ray-project-2.9.1-foss-2023a.eb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        'checksums': ['54cd96e15e1649b75d6c87526a6ff0b6c1b0dd3459f43d9ca11d48c339b68cfc'],
+    }),
+    ('expandvars', '0.12.0', {
+        'checksums': ['7d1adfa55728cf4b5d812ece3d087703faea953e0c0a1a78415de9df5024d844'],
+    }),
+    ('frozenlist', '1.4.1', {
+        'checksums': ['c037a86e8513059a2613aaba4d817bb90b9d9b6b69aace3ce9c877e8c8ed402b'],
+    }),
+    ('Ray', version, {
+        'source_tmpl': '%(namelower)s-%(version)s-cp311-cp311-manylinux2014_x86_64.whl',
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/r/Ray-project/Ray-project-2.9.1-foss-2023a.eb:42]: can you add the wheel for `aarch64`, using the `%(arch)s` template?

**Review by migueldiascosta — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/Ray-project/Ray-project-2.9.1-foss-2023a.eb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    }),
+    ('expandvars', '0.12.0', {
+        'checksums': ['7d1adfa55728cf4b5d812ece3d087703faea953e0c0a1a78415de9df5024d844'],
+    }),
+    ('frozenlist', '1.4.1', {
+        'checksums': ['c037a86e8513059a2613aaba4d817bb90b9d9b6b69aace3ce9c877e8c8ed402b'],
+    }),
+    ('Ray', version, {
+        'source_tmpl': '%(namelower)s-%(version)s-cp311-cp311-manylinux2014_%(arch)s.whl',
+        'checksums': ['fabc520990c1b98dde592813d62737e5e817460e0ac359f32ba029ace292cbe2'],
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/r/Ray-project/Ray-project-2.9.1-foss-2023a.eb:46]:
```suggestion
        'checksums': [{
            '%(namelower)s-%(version)s-cp311-cp311-manylinux2014_x86_64.whl': (
                'fabc520990c1b98dde592813d62737e5e817460e0ac359f32ba029ace292cbe2'
            ),
            '%(namelower)s-%(version)s-cp311-cp311-manylinux2014_aarch64.whl': (
                '1907649d69efdc1b9ffbc03db086f6d768216cb73908ebd4038ac5030effef9e'
            ),
        }],
```

**Review by migueldiascosta — CHANGES_REQUESTED** [review]

**Comment by migueldiascosta** [comment]
@WilleBell I guess the reason for this to be in `foss` instead of `gfbf` is that it can use MPI, but for that doesn't `mpi4py` (https://github.com/search?q=repo%3Aray-project%2Fray%20mpi4py&type=code) need to be included as a dependency? (it's a separate easyconfig now, mpi4py-3.1.4-gompi-2023a.eb)

**Comment by WilleBell** [comment]
> @WilleBell I guess the reason for this to be in `foss` instead of `gfbf` is that it can use MPI, but for that doesn't `mpi4py` (https://github.com/search?q=repo%3Aray-project%2Fray%20mpi4py&type=code) need to be included as a dependency? (it's a separate easyconfig now, mpi4py-3.1.4-gompi-2023a.eb)

Yes, indeed! I added the dependency. Thank you @migueldiascosta

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen3

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
gracehopper - Linux Rocky Linux 9.3, AArch64, ARM UNKNOWN (neoverse_v2), 1 x NVIDIA NVIDIA GH200 480GB, 550.54.15, Python 3.9.18
See https://gist.github.com/migueldiascosta/6ea7aab71e119e43a96b2b260904f3d3 for a full test report.

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @WilleBell!

**Pull request merged**